x-pack/test/security_solution_api_integration/test_suites/entity_analytics/default_license/risk_engine/asset_criticality_privileges.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { ROLES as SERVERLESS_USERNAMES } from '@kbn/security-solution-plugin/common/test';
+import { assetCriticalityRouteHelpersFactoryNoAuth } from '../../utils';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+const USER_PASSWORD = 'changeme';
+const ROLES = [
+  {
+    name: 'security_feature_read',
+    privileges: {
+      kibana: [
+        {
+          feature: {
+            siem: ['read'],
+          },
+          spaces: ['default'],
+        },
+      ],
+    },
+  },
+  {
+    name: 'asset_criticality_index_read',
+    privileges: {
+      elasticsearch: {
+        indices: [
+          {
+            names: ['.asset-criticality.asset-criticality-*'],
+            privileges: ['read'],
+          },
+        ],
+      },
+    },
+  },
+  {
+    name: 'asset_criticality_index_write',
+    privileges: {
+      elasticsearch: {
+        indices: [
+          {
+            names: ['.asset-criticality.asset-criticality-*'],
+            privileges: ['write'],
+          },
+        ],
+      },
+    },
+  },
+];
+
+const ALL_ROLE_NAMES = ROLES.map((role) => role.name);
+
+const allRolesExcept = (role: string) => ALL_ROLE_NAMES.filter((r) => r !== role);
+
+const USERNAME_TO_ROLES = {
+  no_asset_criticality_index_read: allRolesExcept('asset_criticality_index_read'),
+  no_asset_criticality_index_write: allRolesExcept('asset_criticality_index_write'),
+  all: ALL_ROLE_NAMES,
+};
+
+export default ({ getService }: FtrProviderContext) => {
+  describe('@ess privileges_apis', () => {
```

Comment:
More descriptors, IMO:

```suggestion
  describe('@ess Entity Analytics - Asset Criticality Privileges API', () => {
```